files/etc/init.d/elasticsearch.systemd.erb

```diff
@@ -1,8 +1,15 @@
+<%
```

Comment:
This erb interpolation should be encased in `<%- code -%>` rather than `<% code %>` in order to avoid empty newlines in the unit file.